index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Odd Duck</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="css/reset.css">
+    <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+    <header>
+        <h1>Odd Duck Products</h1>
+    </header>
+
+    <main>
+        <!-- create event listener for images and then track some data for market based on clicks and views -->
+        <div class="grid-container">
+            <div class="grid-item">
+                <h2>Box 1. Select your favorite pizza!</h2>
+            </div>
+            <div class="grid-item">Box 2 Image1</div>
```

Comment:
You can do something like this:

       <section>
            <img />
            <img />
            <img />
        </section>
        
To then set global variables in your app.js to display the images there